xml/ha_config_cli.xml

```diff
@@ -94,25 +94,14 @@
   <tip>
    <title>Distinguish Between Shell Prompt and Interactive crm Prompt</title>
    <para>
-    To make all the code and examples more readable, this chapter uses the
-    following notations between shell prompts and the interactive crm
+    To make all the code and examples more readable, this chapter distinguish
+    between the shell root prompt (&prompt.root;) and the interactive crm
     prompt:
    </para>
-   <itemizedlist>
-    <listitem>
-     <para>
-      Shell prompt for user &rootuser;:
-     </para>
-<screen>&prompt.root;</screen>
-    </listitem>
-    <listitem>
-     <para>
-      Interactive &crmsh; prompt (displayed in green, if terminal
-      supports colors):
-     </para>
-<screen>&prompt.crm;</screen>
-    </listitem>
-   </itemizedlist>
+   <screen>&prompt.crm;</screen>
+   <para> For readability reasons, we omit the host names in our books. The
```

Comment:
I would perhaps add one more sentence and slightly rephrase to: 

'By using <command>crm</command> without arguments (or with only one sublevel as argument), the &crmshell; enters the interactive mode. This mode is indicated by the following prompt:

```crm(live/HOSTNAME)# ```
 
For readability reasons, we omit the host name in the interactive crm prompts in our documentation. We only include the host name if you need to run the interactive shell on a specific node, like &node1; for example:

```crm(live/alice)#```

Reply:
Oh, this is soo much better! Thanks! :+1: